src/tcgwars/logic/impl/gen3/PopSeries3.groovy

```diff
@@ -135,14 +142,18 @@ public enum PopSeries3 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 10
+            flip { damage 20 }
           }
         }
         move "Fire Spin", {
           text "70 damage. Discard 2 Basic energy cards attached to Flareon or this attack does nothing."
           energyCost F, F, C
           attackRequirement {}
```

Comment:
I'm in favour of this approach, tho some cards do have the energy available added here I guess?